src/main/java/tds/exam/results/configuration/web/ExamResultsTransmitterApplicationConfiguration.java

```diff
@@ -22,7 +23,8 @@
     CacheConfiguration.class,
     RestTemplateConfiguration.class,
     SecurityConfiguration.class,
-    DataSourceConfiguration.class
+    DataSourceConfiguration.class,
+    EventLoggerConfiguration.class,
```

Comment:
I don't think the extra comma here will allow the Spring container to compile/startup.